app/src/JokeFetch.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import BackButton from "./BackButton";
+import NextButton from "./NextButton";
+
+const buttonText = [
+  "Ugh.",
+  "🤦🏻‍♂️",
+  "omg dad.",
+  "you are the worst",
+  "seriously",
+  "stop it.",
+  "please stop",
+  "that was the worst one",
+];
+
+const JokeFetch = () => {
+  /* Render json from fetchJoke()
```

Comment:
I like the clear structuring in your code, makes it easier to read when closely looking into the comments in between

especially when all those useStates have different purposes